Source/SwiftLintFramework/Extensions/QueuedPrint.swift

```diff
@@ -34,9 +34,12 @@ public func queuedPrint<T>(object: T) {
 A thread-safe, newline-terminated version of fputs(..., stderr).
 
 - parameter string: String to print.
+- parameter silent: True suppresses output
 */
-public func queuedPrintError(string: String) {
-    dispatch_async(outputQueue) {
-        fputs(string + "\n", stderr)
+public func queuedPrintError(string: String, silent: Bool = false) {
```

Comment:
> Put the selective logging logic in the init function which requires no change to logging functions, but makes init more complicated and sets the precedent of distributing logging logic to non-logging objects.

This is what I was trying to say. The logging in Configuration's initializer is already unfortunate but in fact, it should be relatively straightforward to move it to the `swiftlint` CLI app instead, which would address the need to silence some configuration loads without polluting the model object with logging code.

Ideally, all logs would be in `swiftlint` with none in `SwiftLintFramework`.